ironmon_tracker/data/PokemonData.lua

```diff
@@ -447,3774 +396,2966 @@ PokemonData.TypeIndexMap = {
 	[0x11] = PokemonData.Types.DARK,
 }
 
+PokemonData.BlankPokemon = {
+	pokemonID = 0,
+	name = Constants.BLANKLINE,
+	types = { PokemonData.Types.UNKNOWN, PokemonData.Types.EMPTY },
+	abilities = { 0, 0 },
+	evolution = PokemonData.Evolutions.NONE,
+	bst = Constants.BLANKLINE,
+	expYield = 0,
+	movelvls = { {}, {} },
+	weight = 0.0,
+	friendshipBase = 0,
+}
+
 --[[
 Data for each Pokémon (Gen 3) - Sourced from Bulbapedia
 Format for an entry:
+	pokemonID: integer -> The gen 3 pokedex id number for this Pokémon
 	name: string -> Name of the Pokémon as it appears in game
 	types: {string, string} -> Each Pokémon can have one or two types, using the PokemonData.Types enum to alias the strings
 	evolution: string -> Displays the level, item, or other requirement a Pokémon needs to evolve
 	bst: string -> A sum of the base stats of the Pokémon
-	yield: integer -> Base experience yield of the Pokémon
+	expYield: integer -> Base experience yield of the Pokémon
 	movelvls: {{integer list}, {integer list}} -> A pair of tables (1:RSE/2:FRLG) declaring the levels at which a Pokémon learns new moves or an empty list means it learns nothing
 	weight: pokemon's weight in kg (mainly used for Low Kick calculations)
```

Comment:
This "Format for an entry" comment is incorrect:
- `pokemonID`, `types`, and `yield`/`expYield` do not exist in the entries below
- `bst` is no longer a string in the entries below
- `weight` doesn't specify its datatype